products/sle/main.pm

```diff
@@ -743,6 +740,9 @@ sub load_x11tests() {
         if (gnomestep_is_applicable()) {
             loadtest "x11/eog.pm";
             loadtest "x11/rhythmbox.pm";
+            if ((get_var('ADDONS', '') =~ /sdk/ || get_var('SCC_ADDONS', '') =~ /sdk/ || get_var('BUILD_SDK')) && check_var("VERSION", "12-SP1")) {
```

Comment:
And also I don't think it makes sense to only explicitly select one product version. Why should imagemagick only work for 12SP1?